st2common/st2common/transport/connection_retry_wrapper.py

```diff
@@ -126,7 +134,10 @@ def run(self, connection, wrapped_callback):
                 # be notified so raise.
                 if should_stop:
                     raise
+
                 # -1, 0 and 1+ are handled properly by eventlet.sleep
+                self._logger.debug('Received RabbitMQ server error, sleeping for %s seconds '
```

Comment:
Having this log message here would save me a lot of time trying to track down the root cause.